standalone/headers-cache/src/web_api.rs

```diff
@@ -2,27 +2,30 @@ use std::pin::pin;
 
 use anyhow::{bail, Context, Result};
 use log::{debug, error, info};
-use pherry::headers_cache::{read_items_stream, BlockInfo};
+use pherry::{
+    headers_cache::{read_items_stream, BlockInfo},
+    types::Header,
+};
 use rand::Rng;
 use rocket::{
     data::ToByteUnit,
     futures::StreamExt,
-    get,
+    get, put,
```

Comment:
put is not in used?

Reply:
It is used. It is just merged from another `use` statement which cause the diff.